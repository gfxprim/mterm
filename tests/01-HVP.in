5 5
\e[2;2f
