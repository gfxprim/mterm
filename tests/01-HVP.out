 -----
|     |
|     |
|     |
|     |
|     |
 -----
size 5x5 cursor 1x1
